gcd/src/main/java/org/spine3/server/storage/datastore/DsProperties.java

```diff
@@ -162,7 +162,7 @@ static void makeEventFieldProperties(Event event,
         builder.set(producer_id.toString(), producerId);
     }
 
-    static Predicate<Entity> activeEntityPredicate() {
+    static Predicate<Entity> activedEntityPredicate() {
```

Comment:
The point is that this predicate matches the Datastore `Entity`, not any of our types.
As for me, this should stay in `DsProperties` as it's relevant to the Datastore properties.

BTW, let's discuss if we should remove this at all and migrate to using `Query` instead.